ddocs/_users-db/token-login/views/users-by-token/map.js

```diff
@@ -0,0 +1,7 @@
+function(doc) {
+  if (doc.type === 'user' && doc.token_login && doc.token_login.active) {
+    var key = [ doc.token_login.token, doc.token_login.hash ];
+    var value = { token_expiration_date : doc.token_login.expiration_date };
+    emit(key, value);
```

Comment:
When I first wrote this, there was no hash. Added the sha1 hash of the username later on. 
In hindsight, maybe I should encrypt the user's id instead of hashing it so I don't even need this view. 
Y/N?

Reply:
As commented above, I think removing the view and ddoc is a good idea.